schema/method.json

```diff
@@ -12,13 +17,22 @@
             "$ref": "core/primitive/graph.json"
         },
         "units": {
+            "description": "A list of unit methods (containing the method data)",
             "type": "array",
             "items": {
                 "$ref": "method_units/unit_base_referable.json"
             }
+        },
+        "embeddedMethods": {
```

Comment:
Can you clarify the distinction between `embeddedMethods` and `methodGraph`, is it possible to programmatically generate one from the other?

Reply:
To start with, the items in `units` contain the data we want to keep track of (parameters, data, etc.). The field `methodGraph` records how these units relate to each other (for now only connectivity) using flowchartIds as identifiers. Thus, the graph does not contain any information pertaining to the calculation (such as parameters). Finally, `embeddedMethods` is a list of method objects, allowing for the nesting described above. Regarding the programmatic generation: as it is written right now, it would not be possible as the individual units do not include the connectivity information. It could be inferred from the list order, but now we have two lists (one for methods, one for units).

Reply:
![method](https://user-images.githubusercontent.com/10773967/174199533-c3dc2797-a7d4-4ea5-a9f7-ea26bc843520.png)
I tried to draw it 😄

Reply:
I'd just suggest using:

`graph`: ...
`units`:
`methods`:

to clarify and standardize the approach.

Inside workflows, we'd be using:

`graph`
`units`
`subworkflows`
`workflows`

Inside models:

`graph`
`units`
`models`